src/select/__tests__/select.test.tsx

```diff
@@ -560,4 +560,33 @@ describe.each([false, true])('expandToViewport=%s', expandToViewport => {
     const { wrapper } = renderSelect({ autoFocus: true });
     expect(wrapper.findTrigger().getElement()).toHaveFocus();
   });
+
+  test('group options can have description, label tag, tags, disabled reason', () => {
```

Comment:
Great catch!

I added similar tests for select and multiselect but we do not assert disabled reasons for select groups because select groups are not interactive - they can be disabled but the reason is not supported.